powerbi-docs/visuals/power-bi-visualization-export-data.md

```diff
@@ -149,24 +149,26 @@ When you select **Export**, Power BI exports the data to an *.xlsx* file and you
 
-## Protecting proprietary data
+## Customize export data user expirience
 
-Your dataset may have content that should not be seen by all users. If you are not careful, exporting underlying data may let users see all the detailed data for that visual -- every column and every row in the data. 
+Users that granted access to report are **granted access to the entire dataset** unless [row-level security (RLS)](../admin/service-admin-rls.md) limits their access. Report authors and Power BI adminstrators may use capabilities below to customize user experiences when viewing or interacting or export data from reports: 
 
-There are several strategies Power BI admins and designers should use to protect proprietary data. 
+- Report authors [decide which *export options*](#set-the-export-options) are available to users.  
 
-- Designers [decide which *export options*](#set-the-export-options) are available to users.  
-
-- Power BI administrators can turn off data export for their organization. 
+- Power BI administrators can turn off some or all data export options for their organization.  
 
 - Dataset owners can set row level security (RLS). RLS will restrict access to read-only users. But if you have configured an app workspace and given members edit permissions, RLS roles will not be applied to them. For more information, see [Row-level security](../admin/service-admin-rls.md).
 
-- Report designers can hide columns so that they don't show up in the **Fields** list. For more information, see [Dataset properties](../developer/automation/api-dataset-properties.md)
+- Report authors can hide columns so that they don't show up in the **Fields** list. For more information, see [Dataset properties](../developer/automation/api-dataset-properties.md)
+
+
+**These customized user experience do not restrict what data users can access in the dataset. Use [row-level security (RLS)](../admin/service-admin-rls.md) in the dataset so that each person's credentials determine which data they can access.**
 
-- Power BI administrators can add [sensitivity labels](../admin/service-security-data-protection-overview.md) to dashboards, reports, datasets, and dataflows. They can then enforce protection settings such as encryption or watermarks when exporting data. 
+## Protect data when it is exported out of Power BI
 
-- Power BI administrators can use [Microsoft Cloud App Security](../admin/service-security-data-protection-overview.md) to monitor user access and activity, perform real-time risk analysis, and set label-specific controls. For example, organizations can use Microsoft Cloud App Security to configure a policy that prevents users from downloading sensitive data from Power BI to unmanaged devices. 
+- Report auhotrs can apply [sensitivity labels](../admin/service-security-data-protection-overview.md) to reports, upon export to Excel, PowerPoint or PDF file Power BI will apply report's sensitivity label and label's protection settings on the exported file.  
```

Comment:
Report authors should apply

Reply:
Sensitivity label is an opt in optional feature, we can recommend to use it, should might a bit too harsh .

Reply:
to reports. When the report is exported, Power BI will use the sensitivity label to apply the appropriate protection settings to the exported file.  <please add a "for example" here> For example, if the report author has applied the sensitivity label of X, then ABC will happen.